Application/EdFi.Ods.SandboxAdmin/EdFi.Ods.SandboxAdmin.csproj

```diff
@@ -26,7 +26,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="6.0.13" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.13" />
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="6.0.8" />
-    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="6.0.13" />
+    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="6.0.1" />
```

Comment:
Why the downgrade here?

Reply:
10>C:\OSS-Workspace\Ed-Fi-ODS-Implementation\Application\EdFi.Ods.SandboxAdmin\EdFi.Ods.SandboxAdmin.csproj : warning NU1603: EdFi.Ods.Sandbox.Admin depends on Microsoft.Extensions.Caching.Memory (>= 6.0.13) but Microsoft.Extensions.Caching.Memory 6.0.13 was not found. An approximate best match of Microsoft.Extensions.Caching.Memory 7.0.0 was resolved.